### PR TITLE
[TACHYON-1428] Started to document the tachyon.conf and tachyon.exception packages in preparation for the checkstyle rule

### PR DESCRIPTION
**Pull request by HorizonNet**
https://tachyon.atlassian.net/browse/TACHYON-1428

### CODE REVIEW

File: common/src/main/java/tachyon/conf/TachyonConf.java [hunk 1]
```diff
@@ -221,12 +221,25 @@ public void merge(TachyonConf alternateConf) {
   // Public accessor methods
 
   // TODO(binfan): this method should be hidden and only used during initialization and tests.
+
+  /**
+   * Set the value for the appropriate key in the {@link Properties}.
```
Comment (jsimsa) [inline common/src/main/java/tachyon/conf/TachyonConf.java]: Sets

**Comment by jsimsa** [comment]
just one minor comment

**Comment by jsimsa** [comment]
@HorizonNet thanks for the PR, just one minor comment

**Comment by HorizonNet** [comment]
@jsimsa Thanks for the comment. I updated the PR accordingly.

**Comment by jsimsa** [comment]
LGTM

**Pull request merged**